packages/edit-site/src/components/global-styles/typography-panel.js

```diff
@@ -116,6 +120,14 @@ function useHasTextDecorationControl( name, element ) {
 	return ! name && element === 'link';
 }
 
+function useHasTextColumnsControl( name ) {
+	const supports = getSupportedGlobalStylesPanels( name );
+	return (
+		useSetting( 'typography.textColumns', name )[ 0 ] &&
```

Comment:
This line is throwing an error — it looks like after the rebase, this needs to be `useGlobalSetting` now?

Reply:
Thanks for taking a look. I rushed the rebase as I was about to run late for a meeting. I'll fix that up now.

Reply:
✅  Done in https://github.com/WordPress/gutenberg/pull/33587/commits/6a7de1c4f851e13766adec164f8a1cd1a3a0b797

Reply:
The variations path change was also missed after the rebase. I've updated that as well.